src/streamlit_app.py

```diff
@@ -382,327 +59,185 @@ def func_without_errors(*args: typing.Any, **kw: typing.Any) -> typing.Any:
     return func_without_errors
 
 
-def make_link(text: str, entity: str, name: str, val_type: str) -> str:
-    return (
-        f"[{text}](https://app.honeydew.cloud/workspace/{WORKSPACE}/"
-        f"version/{BRANCH}/entity/{entity}/{val_type}/{name})"
-    )
-
-
 @supress_failures
-def get_description(schema: typing.Any, val: str) -> str:
-    entity, name = val.split(".")
-    row_df = schema[(schema["ENTITY"] == entity) & (schema["NAME"] == name)]
-    assert len(row_df) == 1, f"field {val} not found in schema"
-    row = row_df.iloc[0].to_dict()
-    val_type = "metric" if row["TYPE"] == "Metric" else "attribute"
-    display_name = row["DISPLAY_NAME"] if row["DISPLAY_NAME"] is not None else val
-    description = (
-        f'\n\n    {row["DESCRIPTION"]}' if row["DESCRIPTION"] is not None else ""
-    )
-    link = make_link(display_name, entity, name, val_type)
-    return f" * {link}{description}"
-
-
-@supress_failures
```

Comment:
Let's keep the same display format as we had in "explain"

Reply:
take a look at the latest revision